blackbox/build.gradle

```diff
@@ -12,79 +12,114 @@ task bootstrap(type: Exec) {
     commandLine 'sh', "$projectDir/bootstrap.sh"
 }
 
-task unpackDistTar(dependsOn: [project(':app').installDist]) {
+task unpackDistTar(dependsOn: [project(':app').installDist  ]) {
+    println("Copy distribution from " + (project(':app').installCommunityEditionDist.destinationDir) + " to " + crateDist)
     doLast {
         copy {
             from(project(':app').installDist.destinationDir) {
                 includeEmptyDirs = true
             }
             into crateDist
         }
+        lessLogging()
+        ignoreDiskThreshold()
     }
     outputs.dir crateDist
 }
 
-task lessLogging(dependsOn: unpackDistTar) {
+task unpackCommunityEditionDistTar(dependsOn: [project(':app').installCommunityEditionDist]) {
```

Comment:
Lets ensure that we always extract into a fresh directory to ensure that there are no traces left from an older extraction (e.g. CE vs. non-CE installation). Relying that one is calling the `clean` task upfront is not ideal imho.